proxies/nodejs/app.ts

```diff
@@ -1,14 +1,37 @@
-import { initialize } from '@devcycle/nodejs-server-sdk'
+import { DVCClient } from '@devcycle/nodejs-server-sdk'
 import Koa from 'koa'
 import Router from 'koa-router'
+import KoaBody  from 'koa-body'
 
+const data = {
+  clients: {},
+  users: {},
+  commandResults: {}
+}
 
-async function start() {
-    await initialize('<TOKEN>').onClientInitialized()
+const handleClient = async (ctx) => {
+  const body = ctx.request.body
+  if (body.clientId === undefined) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing clientId"
+  } else if (!body.sdkKey) {
+    ctx.status = 400 
+    ctx.body = "Invalid request: Missing sdkKey"
+  } else {
+    const client = new DVCClient(ctx.request.body.sdkKey, ctx.request.body.options)
+    data.clients[ctx.request.body.clientId] = client
+    await client.onClientInitialized()
+
+    ctx.status = 200
+  }
+}
 
+async function start() {
     const app = new Koa()
+    app.use(KoaBody())
```

Comment:
kinda confused at the difference between this and 
https://github.com/koajs/bodyparser